dbus_next/message_bus.py

```diff
@@ -181,6 +186,49 @@ def reply_notify(reply, err):
                     interface='org.freedesktop.DBus.Introspectable',
                     member='Introspect'), reply_notify)
 
+    def emit_interface_added(self, path, interface):
```

Comment:
I don't believe these functions should be public. The `ServiceInterface` is part of the high level api and this detail is taken care of by the library so should never need to be called by the user.

Reply:
That's true, changed.